smalltalksrc/VMMakerTests/MethodBuilderTest.class.st

```diff
@@ -84,6 +84,12 @@ MethodBuilderTest >> testBytecodeAtForMethodWithTwoLiteral [
 	self assert: (memory byteAt: bytecodeAddress) equals: 1
 ]
 
+{ #category : #running }
+MethodBuilderTest >> testClassOfCompiledMethodIsCompiledMethod [
+	
+	self assert: (memory fetchClassOf: methodBuilder newMethod buildMethod) equals: (memory splObj: 16).
```

Comment:
I think this test is simple and neat.
The methods created by the method builder should have the correct class.